src/TorchSharp/Optimizers/Rprop.cs

```diff
@@ -86,7 +86,7 @@ public class Rprop : OptimizerHelper
             /// <param name="min_step">Minimum allowed step size.</param>
             /// <param name="max_step">Maximum allowed step size.</param>
             /// <param name="maximize">Maximize the params based on the objective, instead of minimizing.</param>
-            public Rprop(IEnumerable<Parameter> parameters, double lr = 0.01, double etaminus = 0.5, double etaplus = 1.2, double min_step = 1e-6, double max_step = 50, bool maximize = false)
+            public Rprop(IEnumerable<Parameter> parameters, double lr = 1e-2, double etaminus = 0.5, double etaplus = 1.2, double min_step = 1e-6, double max_step = 50, bool maximize = false)
```

Comment:
Isn't 0.01 and 1e-2 the same value?

Reply:
Yes it's the same, I modified it because everywhere else it was using the 1e-2 notation